packages/bpk-component-select/readme.md

```diff
@@ -19,7 +19,7 @@ export default () => (
     id="fruits"
     name="fruits"
     value="oranges"
-    onChange={() => console.log('select changed')}
+    onChange={(e) => console.log(`select changed to ${e.target.value}`)}
```

Comment:
makes evident what you receive in the callback, I though it was directly the value